core/pom.xml

```diff
@@ -419,7 +419,7 @@ THE SOFTWARE.
     <dependency><!-- groovy shell uses this but it doesn't declare this dependency -->
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>1.11</version>
+      <version>1.14</version>
```

Comment:
@ssbarnea I would recommend to do the upgrade in a separate PR. Commonly in such PRs we provide links to available changelogs and the full code diff.